### DISCUSSION TITLE
Is there some kind of http api?

### DISCUSSION THREAD

**Discussion by michabbb** [Ironclad/rivet#104] [Q&A] [answered]
Thanks

**Comment by abrenneke** [comment]
Hey, do you mean like, a node that can make arbitrary HTTP calls?

**Reply by michabbb** [reply]
I am mean, whatever I build with the gui, is it possible to use that agent via a http api, so that I can use the agent for example with php?

**Reply by abrenneke** [reply, marked as answer]
Got it, no there's nothing like that built in to Rivet! You'll have to make a small HTTP API in TypeScript or JavaScript in order to call your Rivet graph over HTTP. Check out https://github.com/Ironclad/rivet-example for an example app that does this!

**Reply by michabbb** [reply]
okay, thanks for your feedback !
